packages/ui/src/components/support/SupportHeroComponent.tsx

```diff
@@ -0,0 +1,32 @@
+export interface SupportHeroProps {
+  title?: string
+  subtitle?: string
+  backgroundImage?: string
+}
+
+const SupportHeroComponent = ({ backgroundImage, subtitle, title }: SupportHeroProps) => {
```

Comment:
This component is not only for Support page but a general `Hero` component. I would create a component in `packages/ui/src/components/Hero/Hero.tsx`

Reply:
You can also add `Hero.stories.tsx` in the same folder